src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -215,6 +215,12 @@ public partial class View
         public static readonly BindableProperty BackgroundImageProperty = BindableProperty.Create(nameof(BackgroundImage), typeof(string), typeof(View), default(string),
             propertyChanged: (bindable, oldValue, newValue) =>
             {
+                if (oldValue == newValue || String.Equals(oldValue, newValue))
```

Comment:
it seems that String.Equals(oldValue, newValue) is enough

Reply:
thank you for review. I have changed as following your comment.